lib/SelectWallet.js

```diff
@@ -139,7 +205,10 @@ class SelectWallet extends Component {
               colHeaders={['Accounts']}
               styles={styles.selectListStyle}
               tableData={accountTableData}
-              onRowClick={e => handleSelect(e.rowData.Accounts, 'account')}
+              onRowClick={e => {
```

Comment:
Do we need to catch the `event` and run `preventDefault` here?

Reply:
if so, i have a function that runs `preventDefault` on the `event` and then returns the `event` so it can be used like so:

```
preventDefault(e).rowData.Accounts
```

Reply:
cool utility! You should definitely add these to bpanel-utils